flaskfront/opgflaskfront/templates/feedback.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+
+{%- from 'govuk_frontend_jinja/components/inset-text/macro.html' import govukInsetText -%}
+{% block pageTitle %}Send us feedback (flask){% endblock %}
```

Comment:
Remove "(flask)"

Reply:
Bear in mind this will shortly be replaced by an actual form,  but.. done :)